client/post-editor/media-modal/test/index.jsx

```diff
@@ -323,6 +323,22 @@ describe( 'EditorMediaModal', () => {
 		expect( buttons[ 1 ].label ).to.be.equals( 'Insert' );
 	} );
 
+	test( 'should show an insert button if multiple images are selected when gallery view is disabled', () => {
```

Comment:
Nice, thanks for adding this test case - should we also have one for the default prop to ensure the gallery flow is still functional with the "Continue" button?

Reply:
I think there's an existing case that covers the 'Continue' button for the default value, but I've added in another for when the galleryViewEnabled prop is explicitly set to true as well.

https://github.com/Automattic/wp-calypso/blob/master/client/post-editor/media-modal/test/index.jsx#L301-L314